include/pybind11/detail/internals.h

```diff
@@ -39,7 +39,6 @@
 /// further ABI-incompatible changes may be made before the ABI is officially
 /// changed to the new version.
 #ifndef PYBIND11_INTERNALS_VERSION
-//   REMINDER for next version bump: remove loader_life_support_tls
 #    define PYBIND11_INTERNALS_VERSION 11
```

Comment:
We could make the default version dependent on the Python version again (we did this before). Concretely, if we detect Python >= 3.14, `#define PYBIND11_INTERNALS_VERSION 12`. That'll add to the test coverage without breaking any existing ABIs.